allennlp/predictors/open_information_extraction.py

```diff
@@ -60,6 +60,116 @@ def make_oie_string(tokens: List[Token], tags: List[str]) -> str:
 
     return " ".join(frame)
 
+def get_predicate_indices(tags: List[str]) -> List[int]:
+    """
+    Return the word indices of a predicate in BIO tags.
+    """
+    return [ind for ind, tag in enumerate(tags) if 'V' in tag]
+
+def get_predicate_text(sent_tokens: List[Token], tags: List[str]) -> str:
+    """
+    Get the predicate in this prediction.
+    """
+    return " ".join([sent_tokens[pred_id].text
+                     for pred_id in get_predicate_indices(tags)])
+
+def predicates_overlap(tags1: List[str], tags2: List[str]) -> bool:
+    """
+    Tests whether the predicate in BIO tags1 are subsumed in
+    those of tags2.
+    """
+    # Get predicate word indices from both predictions
+    pred_ind1 = get_predicate_indices(tags1)
+    pred_ind2 = get_predicate_indices(tags2)
+
+    # Return if pred_ind1 is contained in pred_ind2
+    return any(set.intersection(set(pred_ind1), set(pred_ind2)))
```

Comment:
this returns true if they intersect, is that ok?

Reply:
I think it's ok, maybe just the docstring and comment are wrong?

Reply:
Yes, intersection is what I was going for.  The docstrings were wrong.